testUtils/mockHttpCalls/securityv1alpha1/organization.ts

```diff
@@ -5,4 +5,7 @@ export const getOrganizationByName = {
     name: 'org1',
   },
   spec: {},
+  status: {
+    namespace: 'org-org1',
+  },
```

Comment:
I think the test case would be even more meaningful if the actual namespace value would be different from the pattern `org-` + org name.

Reply:
I think the test for `ClusterDetail` should be testing whether the namespace returned by the first mocked response can be used for getting the second response. Since they're both being mocked for the test, it would make more sense to test that, rather than the specific output.

Thanks for pointing this out - I'll make the changes for the test!